src/test/java/com/exasol/hadoop/HdfsSerDeExportServiceTest.java

```diff
@@ -205,13 +254,25 @@ public void testExportParquetString() throws Exception {
         columns.add(new HCatTableColumn("v1", "varchar(1)"));
         columns.add(new HCatTableColumn("v2", "varchar(10)"));
         columns.add(new HCatTableColumn("s1", "string"));
+        columns.add(new HCatTableColumn("charnull", "string"));
```

Comment:
The dataType should probably be char(X).

Reply:
Sorry, my bad - copy&paste error. Will fix this.